wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -330,7 +331,6 @@ public void onChanged(Resource<Boolean> status) {
                 showHideJoinDashPayAction();
             }
         });
-        dashPayViewModel.isPlatformAvailable();
```

Comment:
I am curious about this change.  How do we trigger this call?

Reply:
It is called automatically by the SDK when you starting observing on it:
`dashPayViewModel.isPlatformAvailableLiveData().observe(...)`